google-cloud-bigtable/src/test/java/com/google/cloud/bigtable/admin/v2/it/BigtableBackupIT.java

```diff
@@ -360,6 +362,55 @@ public void backupIamTest() throws InterruptedException {
     assertThat(permissions).hasSize(4);
   }
 
+  @Test
```

Comment:
FYI, this will need to be rebased when #498 is merged

Reply:
Rebase the test code.